paper/paper.md

```diff
@@ -0,0 +1,78 @@
+
+## Features
+The OpenTURNS library provides functions for defining a probabilistic model for a random vector or stochastic process. A multivariate distribution can be defined in a number of ways: directly, by decomposition into marginals and copulas, or by conditioning. For a one-dimensional distribution and a given function, we can determine the push-forward distribution. In addition, the library can be used to define functions acting on these objects, and to combine these functions. The calculation of the gradient or hessian of a function is taken into account. In the context of uncertainty quantification, the library provides services for sampling the law of the input random vector to the physical model and propagating uncertainties to the output. Sampling can be performed using Monte-Carlo, quasi Monte-Carlo and variance reduction methods (importance sampling, line sampling). Different types of experimental designs are implemented including Latin Hypercube Sampling (LHS) and optimized LHS.
+
+The library can also be used to evaluate statistics from a sample. We can estimate the parameters of a probabilistic model including extreme value models (e.g. Generalized Extreme Value distribution and Generalized Pareto distribution) and validate them using fitting tests (e.g. Kolmogorov-Smirnov, Anderson-Darling, etc.) or using graphical tests (e.g. QQ-plot, Kendall plot). Non parametric models can be used based on a sample including kernel density estimation or Bernstein copula. For a given model, variables can be prioritized using sensitivity analysis methods (e.g. SRC, Sobol' and HSIC indices). A function can also be approximated by a meta-model, including linear model, polynomial chaos expansion, Gaussian process regression. A unified interface of optimization algorithms (NLOpt, Ipopt, PAGMO, Bonmin, Dlib, Ceres, Cminpack, Cobyla, etc.) is available and used in particular in the context of parametric model calibration with Bayesian or non-Bayesian methods. Since many probabilistic methods require the evaluation of an integral, numerical integration methods are available by sampling or quadrature (if necessary, adaptive). The provided quadrature methods are the multivariate Gaussian quadrature (e.g. Gauss Legendre quadrature), Gauss-Kronrod, Smolyak quadrature and other algorithms including algorithms from Cuba's library.  The library provides full or compressed linear algebra capabilities (in particular for covariance matrices). Numerous reliability algorithms are implemented for probability calculation and quantification of rare events (e.g. FORM-SORM, subset sampling). Other reliability algorithms such as Non parametric Adaptive Importance Sampling (NAIS) and Cross-Entropy are available.
+
+## Usages
```

Comment:
Maybe in Usages (or somewhere else in the paper) it could be valuable to state that the release cycle is twice per year ?